devenv/docker/blocks/influxdb1/docker-compose.yaml

```diff
@@ -22,11 +22,11 @@
       FD_PORT: 8086
 
   telegraf-influxdb1:
-    image: telegraf:1.19.1
+    image: telegraf:latest
```

Comment:
Will latest telegram be always compatible with Influx 1?

Reply:
@ifrost not sure, but in this case i think it will be ok: in grafana we are using the output plugin named "outputs.influxdb" for influx1 and the plugin named "outputs.influxdb_v2" for influx2, so i hope that should hold.